dev_guide/deployments.adoc

```diff
@@ -247,7 +247,7 @@ triggers:
 === Image Change Trigger
 
 The ImageChange trigger results in a new deployment whenever the value of an
-image stream tag changes.
+image stream tag changes either by a build or because it was imported.
```

Comment:
> @bparees do you mean that a "ConfigChangeTrigger" need to exist for an "ImageChange" trigger to function?

Oh yeah looking at the table and other comments that seems to be indeed the case... This explains some puzzling behavior @bbguimaraes was observing with the logging templates lately!